.eslintrc-md.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Ready to move up to lint-all?

Reply:
Done and fixed all new linting errors (mostly CRs at the ends of files).